### PR TITLE
added skills and footer section

### PR DESCRIPTION
**Pull request by Tugeeta**

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	// GET HTML ELEMENT BY USING DOM
+	const skillsSection = document.getElementById('#skillSec');
+	const orderedList = document.querySelector('ol');
+    for (let i = 0; i < skillsArray.length; i++) {
+		 
+        // CREATION OF LIST ITEM THROUGH DOM
+        const listItem = document.createElement('li');
+        listItem.innerHTML = skillsArray[i];
+	orderedList.appendChild(listItem);
+     }
```
Comment (mccalluc) [inline js/index.js:30]: Consistent indentation will make it more readable.

File: js/index.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+
+//SELECTING FOOTER AND PARAGRAPH ELEMENT USING DOM
+
+const today = new Date();
+const thisYear = today.getFullYear();
+
+
+const Footer = document.querySelector('footer');
```
Comment (mccalluc) [inline js/index.js:8]: better to use lower case, unless you're defining a class.

File: index.html [hunk 3]
```diff
@@ -155,13 +155,31 @@
...
        text-shadow:2px 1px yellow;
+       padding: 15px;
      } 
+     ol li{
+ list-style:none;
+ }
+#skillSec li{
+display:inline-block;
+width:200px;
+height:100px;
```
Comment (mccalluc) [inline index.html:166]: This may be brittle -- If you have text that is longer than expected, what will happen?

**Comment by Tugeeta** [comment]
Good Evening Mr. McCallum,

Thank you for your input and suggestions.  I will make changes, but codes
work differently in different browsers.  In Chrome, I increased the width:
200px; height: 100px; whereas in internet explorer width:115 px; and
height: 65px.

Once again thank you very much for your input!
Have a nice evening!
Geeta Turumella

**Pull request merged**